app/controllers/base_controller.rb

```diff
@@ -1,6 +1,15 @@
 class BaseController < ApplicationController
   include Flowable
 
+  # This stops the browser caching these pages.
+  # This is done so that someone can't use the Back button to return to a users pages
+  # after they have logged out. There is a cost to page load speeds as a result
+  def set_cache_buster
```

Comment:
This method still seems to be only called by the providers controllers but not the citizens controllers.
Is this an oversight or the intended use?